src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const defaultCompare = (previous, current) => {
+  const stringPrevious = String(previous);
+  const stringCurrent = String(current);
+
+  switch (true) {
+    case stringPrevious > stringCurrent:
+      return 1;
+
+    case stringPrevious < stringCurrent:
+      return -1;
+  }
+};
```

Comment:
Why you use switch statement, if  you have 2 variants to return. Use:
  return (...condition)
    ? ...value for true
    : ...value for false.

Reply:
Fixed ✔